src/DotNetLightning.Core/Serialization/LightningStream.fs

```diff
@@ -325,20 +325,22 @@ type LightningReaderStream(inner: Stream) =
         let len = this.ReadUInt16(false)
         this.ReadBytes(int32 len)
 
+    member this.ReadKey() =
```

Comment:
should this rather be called ReadPrivKey()?

Reply:
I named it `ReadKey` because the NBitcoin type (which it returns) is named `Key`. Though I agree the lack of consistency between "Key" and "PrivKey" is bad.

I've added a type annotation to the method (and all the other methods of `LightningStream`) to make this slightly clearer.